proc/PrepSyncToKernel.bat

```diff
@@ -0,0 +1,59 @@
+# Prepara Sincronizacao com Kernel
+# Copia Scilista para diretorio do Escalonador
+# Copia Pacotes da lista para diretorio do Escalonador
+
+# SCILISTA_PATH: path do arquivo scilista que sera usado no processamento iniciado pelo GeraPadrao
+# XC_SPS_PACKAGES: path do diretório com todos os pacotes gerados pelo XC
+# XC_KERNEL_GATE: path do diretório para copia dos pacotes como estao no momento que o processamento do GeraPadrao e iniciado
+
+
+echo ===============
+echo
+echo "Prepara Sincronizacao com o Kernel..."
+echo "Copiando pacotes SPS para a área do Escalonador"
+echo
+echo ===============
+
+if [ ! -z ${SCILISTA_PATH+x} ] && [ ! -z ${XC_SPS_PACKAGES+x} ] && [ ! -z ${XC_KERNEL_GATE+x} ];
+then
+    while read LINE; do
+        ACRON="$(echo $LINE | cut -f1 -d ' ')"
+        ISSUE="$(echo $LINE | cut -f2 -d ' ')"
+        DEL_COMMAND="$(echo $LINE | cut -f3 -d ' ')"
+        echo
+        echo "ACRON: $ACRON"
+        echo "ISSUE: $ISSUE"
+        echo
+
+        if [[ $(tr '[:upper:]' '[:lower:]' <<< "$DEL_COMMAND") = del ]];
```

Comment:
@patymori, porque os pacotes marcados para `deleção` não são copiados?

Reply:
@joffilyfe, até onde eu sei esse comando serve para indicar aos scripts do proc que este fascículo deve ser apagado. No caso do *SPF*, temos que fazer essa deleção no Airflow e tem um ticket para fazer isso: https://github.com/scieloorg/opac-airflow/issues/56.
@robertatakenaka, talvez você consiga explicar com mais detalhes do que eu, pois não saberia dizer exatamente os scripts do proc fazem

Reply:
Isso quer dizer que as linhas que indicam os pacotes não possuirão o comando `del`? Se for assim então tudo bem.

Reply:
O que estiver indicado com o comando `del` na scilista não terá pacotes necessariamente no diretório e, por isso, não teria que ser copiado. É por isso que, sempre que há necessidade de apagar os documentos de um fascículo para substituir por outros, a scilista viria assim:
```
rsp v12 del
rsp v12
```